kernel/Kconfig

```diff
@@ -249,6 +249,7 @@ config DYNAMIC_THREAD_STACK_SIZE
 	default 4096 if X86
 	default 1024 if !X86 && !64BIT
 	default 2048 if !X86 && 64BIT
+	default 4096 if COVERAGE
```

Comment:
Defaults are picked up on a first match basis, I think this needs to be up the top, unless you want the stack size to be 1024 when `!X86 && !64BIT && COVERAGE`